src/providers/wallet_daemon/provider.ts

```diff
@@ -22,7 +22,6 @@ import {
 import {WebRtcRpcTransport} from "./webrtc_transport";
 
 export const WalletDaemonNotConnected = 'WALLET_DAEMON_NOT_CONNECTED';
-export const Unsupported = 'UNSUPPORTED';
```

Comment:
I'm not convinced we should be deleting this line in this PR.
Can we revert changes to files under src/providers/wallet_daemon ?

Reply:
It wasn't used anywhere but since this PR is about only Tari Universe provider I have reverted this change